nncf/openvino/optimized_functions/models.py

```diff
@@ -381,6 +387,7 @@ def _build_compress_model(
     # Build OV model
     weight = opset.parameter(weight_shape, name="weight", dtype=DTYPE_MAP_OV[weight_dtype])
     ov_parameters = [weight]
+    weight = convert_op(weight, ov.Type.f32)
```

Comment:
Casting to FP32 in the beginning because FP8 data type does not support `abs` operation.